Std/Data/List/Basic.lean

```diff
@@ -1676,4 +1676,14 @@ See `isSubperm_iff` for a characterization in terms of `List.Subperm`.
 -/
 def isSubperm [BEq α] (l₁ l₂ : List α) : Bool := ∀ x ∈ l₁, count x l₁ ≤ count x l₂
 
-end List
+/--
+`O(|l|)`. Inserts `a` in `l` right before the first element such that `p` fails, or at the end of
+the list if `p` never fails on `l`.
```

Comment:
Before reading the docs, my initial guess was that the polarity of `p` would be flipped, and it inserts `a` before the first element for which `p` is true. This seems more consistent with `findP` et al.